packages/react/src/components/DetailsList/DetailsRow.base.tsx

```diff
@@ -327,7 +327,7 @@ export class DetailsRowBase extends React.Component<IDetailsRowBaseProps, IDetai
         className={this._classNames.root}
         data-selection-index={itemIndex}
         data-selection-touch-invoke={true}
```

Comment:
Maybe this doesn't matter but it's also not possible for users to override this attribute. Seems similar to the one you changed.